packages/pluginsdk/package.json

```diff
@@ -31,67 +31,17 @@
     "yargs": "^16.0.3"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "15.0.0",
+    "@rollup/plugin-commonjs": "15.1.0",
     "@rollup/plugin-node-resolve": "9.0.0",
     "@rollup/plugin-typescript": "6.0.0",
-    "@spinnaker/core": "0.0.507",
-    "rollup": "2.27.1",
+    "@spinnaker/core": "0.0.517",
+    "@types/yargs": "^15.0.7",
+    "rollup": "2.28.2",
     "rollup-plugin-external-globals": "0.6.0",
     "rollup-plugin-less": "1.1.2",
     "rollup-plugin-postcss": "3.1.8",
-    "typescript": "4.0.2"
+    "typescript": "4.0.3"
   },
-  "peerDependencies": {
-    "@rollup/plugin-commonjs": "15.0.0",
-    "@rollup/plugin-node-resolve": "9.0.0",
-    "@rollup/plugin-typescript": "6.0.0",
-    "@spinnaker/core": "0.0.507",
-    "@spinnaker/eslint-plugin": "1.0.7",
-    "@types/react": "16.8.25",
-    "@typescript-eslint/eslint-plugin": "2.26.0",
-    "@typescript-eslint/parser": "2.26.0",
-    "@uirouter/core": "6.0.4",
-    "@uirouter/react": "1.0.2",
-    "eslint": "6.8.0",
-    "eslint-config-prettier": "6.10.1",
-    "eslint-plugin-react-hooks": "3.0.0",
-    "husky": "4.2.5",
-    "lodash-es": "4.17.15",
-    "npm-run-all": "4.1.5",
```

Comment:
These are the previous peerDependencies that got moved into `pluginsdk-peerdeps`